README.md

```diff
@@ -159,6 +169,48 @@ directory to export a version-specific property:
+`defects4j query -p Chart -q "report.id,report.url" will provide the a list of
+all active bug IDs, along with the bug report ID and bug report URL for each.
+
+
+| Property              | Description                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------|
+| bug.id                | Assigned bug IDs (included in all results)                                          |
+| project.id            | Assigned project ID                                                                 |
+| project.name          | Original repository name                                                            |
+| revision.id.buggy     | Commit hashes for the buggy version of each bug                                     |
+| revision.id.fixed     | Commit hashes for the fixed version of each bug                                     |
+| revision.buggy.date   | Date of the buggy commit for each bug                                               |
+| revision.fixed.date   | Date of the fixed commit for each bug                                               |
+| report.id             | Bug report ID from the version tracker for each bug                                 |
+| report.url            | Bug report URL from the version tracker for each bug                                |
+| build.file            | Location of the Defects4J build file for the project                                |
+| vcs                   | Version control system used by the project                                          |
+| repository            | Location of the project repository                                                  |
+| commit.db             | Location of the CSV containing information on that bug                              |
+| classes.modified      | Classes modified by the bug fix                                                     |
+| classes.relevant.src  | Source classes loaded by the JVM when executing all triggering tests                |
+| classes.relevant.test | Test classes loaded by the JVM when executing all triggering tests                  |
+| tests.relevant        | List of relevant tests classes (a test class is relevant if, when executed, the JVM loads at least one of the modified classes) |
+| tests.trigger         | List of test methods that trigger (expose) the bug                                  |
+| tests.trigger.cause   | List of test methods that trigger (expose) the bug, along with the root cause       |
+| deprecated.version    | (for deprecated bugs only) Version of Defects4J where a bug was deprecated          |
+| deprecated.reason     | (for deprecated bugs only) Reason for deprecation                                   |
+
+By default, `defects4j query` returns information on active bugs. The `[-d]`
+flag returns information only on deprecated bugs, while the `[-a]` flag returns
```

Comment:
Usually, flags are lower-case if they take an argument and upper-case if they don't. I think these should be -D and -A.

Reply:
Corrected in latest commit.